django/conf/global_settings.py

```diff
@@ -666,3 +666,8 @@ def gettext_noop(s):
 SECURE_REFERRER_POLICY = "same-origin"
 SECURE_SSL_HOST = None
 SECURE_SSL_REDIRECT = False
+
+###################
+# GZIP MIDDLEWARE #
+###################
+GZIP_MIDDLEWARE_RANDOM_BYTES = 100
```

Comment:
Playing devil's advocate for a moment here, and to ensure the discussion is had, do we really _need_ a new setting here? 
Assuming `100` is enough in the general case[^1] and is a _good_ default, we could also possibly consider having it be directly on the `GZipMiddleware` class and letting downstream users subclass that if they wish to change the value[^2]. Though there might be considerations around other related concepts, like using it via `decorator_from_middleware` which I've not given a second thought to...

[^1]: I've skimmed the paper to understand where that value came from, but I'm not exactly used to reading papers, so there may be nuance that I've missed.
[^2]: I wonder, under what scenarios one might wish to change it? Any ideas?

Reply:
Totally agree, the number of random bytes should be a `GZipMiddleware` attribute.

Reply:
Good idea, I agree, using an attribute is much better!

Regarding picking the number, I picked 100 because it was the highest (safest) number discussed the paper. I have reached out to the paper authors to get their opinion on a good number being default in Django but has not gotten any feedback from them (yet).

Reply:
I think the only downside of using the highest number is the overhead of sending more bytes. However, the extra 50 bytes (average) does not feel that excessive to me. Especially since the other option may in many cases be to disable gzip altogether.